variables.tf

```diff
@@ -244,6 +244,46 @@ variable "containers_config_map" {
       nvme     = 8
       nics     = 8
       memory   = ["357GB", "384GB"]
+    },
+    Standard_L8as_v3 = {
+      compute  = 1
+      drive    = 1
+      frontend = 1
+      nvme     = 1
+      nics     = 4
+      memory   = ["29GB", "29GB"]
+    },
+    Standard_L16as_v3 = {
+      compute  = 4
+      drive    = 2
+      frontend = 1
+      nvme     = 2
+      nics     = 8
+      memory   = ["72GB", "73GB"]
+    },
+    Standard_L32as_v3 = {
+      compute  = 4
+      drive    = 2
+      frontend = 1
+      nvme     = 4
+      nics     = 8
+      memory   = ["190GB", "190GB"]
+    },
+    Standard_L48as_v3 = {
+      compute  = 3
+      drive    = 3
+      frontend = 1
+      nvme     = 6
+      nics     = 8
+      memory   = ["308GB", "308GB"]
+    },
+    Standard_L64as_v3 = {
+      compute  = 4
+      drive    = 2
+      frontend = 1
+      nvme     = 8
+      nics     = 8
+      memory   = ["425GB", "425GB"]
```

Comment:
max is `memory   = ["357GB", "384GB"]`

Reply:
` python3 gen.py --net eth0 --compute-dedicated-cores 4 --drive-dedicated-cores 2 --frontend-dedicated-cores 1
INFO:resources generator:Drives to be allocated: [{'path': '/dev/nvme0n1'}, {'path': '/dev/nvme1n1'}, {'path': '/dev/nvme2n1'}, {'path': '/dev/nvme3n1'}, {'path': '/dev/nvme4n1'}, {'path': '/dev/nvme5n1'}, {'path': '/dev/nvme6n1'}, {'path': '/dev/nvme7n1'}]
INFO:resources generator:NUMA 0 cores: [56, 24, 40, 32, 50, 16, 22, 30, 12, 4, 48, 14, 38, 18, 44, 20, 46, 52, 6, 60, 26, 8, 2, 42, 62, 36, 54, 28, 10, 58, 34], total memory: 503.63648986816406 GiB
INFO:resources generator:num_containers_by_role[FRONTEND]: 1, nodes count: 1
INFO:resources generator:num_containers_by_role[DRIVES]: 1, nodes count: 2
INFO:resources generator:num_containers_by_role[COMPUTE]: 1, nodes count: 4
INFO:resources generator:Added FRONTEND container resources
INFO:resources generator:Added DRIVES container resources
INFO:resources generator:Added COMPUTE container resources
INFO:resources generator:7 io nodes on NUMA 0
INFO:resources generator:allocating 424.8637130744755 GiB for COMPUTE container, (4 nodes)
weka@test2:~$ python3 gen.py --net eth0 --compute-dedicated-cores 5 --drive-dedicated-cores 2 --frontend-dedicated-cores 0
INFO:resources generator:Drives to be allocated: [{'path': '/dev/nvme0n1'}, {'path': '/dev/nvme1n1'}, {'path': '/dev/nvme2n1'}, {'path': '/dev/nvme3n1'}, {'path': '/dev/nvme4n1'}, {'path': '/dev/nvme5n1'}, {'path': '/dev/nvme6n1'}, {'path': '/dev/nvme7n1'}]
INFO:resources generator:NUMA 0 cores: [18, 48, 20, 4, 14, 32, 42, 12, 10, 52, 28, 58, 62, 6, 22, 44, 56, 54, 24, 26, 40, 38, 60, 50, 30, 34, 46, 8, 2, 36, 16], total memory: 503.63648986816406 GiB
INFO:resources generator:num_containers_by_role[FRONTEND]: 0, nodes count: 0
INFO:resources generator:num_containers_by_role[DRIVES]: 1, nodes count: 2
INFO:resources generator:num_containers_by_role[COMPUTE]: 1, nodes count: 5
INFO:resources generator:Added DRIVES container resources
INFO:resources generator:Added COMPUTE container resources
INFO:resources generator:7 io nodes on NUMA 0
INFO:resources generator:allocating 424.7984994156286 GiB for COMPUTE container, (5 nodes)`

Reply:
`python3 gen.py --net eth0 --compute-dedicated-cores 5 --drive-dedicated-cores 2 --frontend-dedicated-cores 0
INFO:resources generator:allocating 424.7984994156286 GiB for COMPUTE container, (5 nodes)`